### DISCUSSION TITLE
HomeAssistant

### DISCUSSION THREAD

**Discussion by SKranny** [arendst/Tasmota#16698] [General]
Hello, I need integration for Home Assistant, with which you can change the ssid and password on Mikrotik in the Web interface, and then on Tasmota Sonoff devices connected to Mikrotik. Tell me, please, how can this be done?

**Comment by barbudor** [comment]
Not sure to understand what you mean
You can change Tasmota Wifi settings using commands `SSID` and `PASSWORD`
You send that:
- In the console : `Backlog SSID xxxxx; PASSWORD xxxxx`
- Through serial with the same
- Through MQTT on `topic cmnd/%topic%/backlog` and payload `SSID xxxxx; PASSWORD xxxxx`

**Reply by sfromis** [reply]
And with a GroupTopic (default for Tasmtoa devices is `tasmotas`), with one single command you can reach all available Tasmota devices.
Topic would then normally be: `cmnd/tasmotas/backlog`
